src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -351,19 +373,31 @@ private function getMatches(string $line, bool $matchCommentOnly = false): ?arra
                 $matches['tag'] = $matches['tag3'];
                 $matches['hint'] = $matches['hint3'];
                 $matches['var'] = $matches['signature'];
+
+                // Since static can be both a return type declaration & a keyword that defines static methods
+                // we assume it's a type declaration when only one value is present
+                if ('' === $matches['hint'] && '' !== $matches['static']) {
+                    $matches['hint'] = $matches['static'];
+                    $matches['static'] = '';
+                }
```

Comment:
I think this is not covered by added tests.

Reply:
@julienfalque Hi, this case is in fact covered by line 1116 https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/63740ece1ea08855db4895d9c8b5d2e904b1e9fc/tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php#L1111-L1122

Reply:
Sorry, I meant a case with `@return static` (or maybe both).

Reply:
Sorry, I'm not sure what you mean, the problem with `static bar(string $foo)` is that it's ambiguous, it could be a static method that returns `mixed` or a non static method that returns `static`. Since there is no way of knowing which one it is, we assume it's the latter. As for both, this case is covered by lines 1117-1120.

Reply:
I would like to have a test case that shows the fixer behaves as expected when encountering a `@return static`. For example:
```diff
 /**
- * @param string $foo
+ * @param  string $foo
  * @return static
  **/
```

Reply:
Done https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6366/commits/3274c1c44b5294353efafc747b003ceb899d8fc6